confluence-xml/src/test/resources/confluencexml/include/entities.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hibernate-generic>
+  <object class="Space" package="com.atlassian.confluence.spaces">
+    <id name="id">100</id>
+    <property name="name"><![CDATA[TestSpace]]></property>
+    <property name="key"><![CDATA[TestSpace]]></property>
+  </object>
+
+  <object class="Page" package="com.atlassian.confluence.pages">
+    <id name="id">10</id>
+    <property name="space" class="Space" package="com.atlassian.confluence.spaces">
+      <id name="id">100</id>
+    </property>
+    <property name="title"><![CDATA[TestContent]]></property>
+    <property name="parent" class="Page" package="com.atlassian.confluence.pages">
+      <id name="id">12</id>
+    </property>
+    <collection name="bodyContents">
+      <element class="BodyContent" package="com.atlassian.confluence.core">
+        <id name="id">0</id>
+      </element>
+    </collection>
+  </object>
+
+  <object class="BodyContent" package="com.atlassian.confluence.core">
+    <id name="id">0</id>
+    <property name="body">
+      <![CDATA[
+          <p>This page includes another page:</p>
+          <ac:structured-macro ac:name="include" ac:schema-version="1" data-layout="default" ac:local-id="0c4cd4d7-167b-455f-aacc-d793ea14616a" ac:macro-id="e2cca8926332c500394f85da339d3bf0">
+            <ac:parameter ac:name="">
+                <ac:link>
+                    <ri:page ri:content-title="Includee" ri:version-at-save="1" />
+                </ac:link>
+            </ac:parameter>
+          </ac:structured-macro>
+          <p>Additional includes from the JIRA ticket:</p>
```

Comment:
I would remove that paragraph as it does not really bring much to the validated use case.